aunt-leahs-app/src/components/checkbox.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import {
+  Typography,
+  Checkbox,
+  FormControlLabel,
+  makeStyles
+} from "@material-ui/core";
+import "../styles.css";
+
+const useStyles = makeStyles({
+  root: {
+    color: "#00A19B", // If unspecified, default color is primary
+    "&$checked": {
+      color: "#00A19B"
+    }
+  },
+
+  colorPrimary: {
+    "&$checked": {
+      color: "#00A19B",
+      "&:hover": {
+        backgroundColor: "transparent",
+        "@media (hover: none)": {
+          backgroundColor: "transparent"
+        }
+      }
+    },
+    "&:hover": {
+      backgroundColor: "transparent",
+      "@media (hover: none)": {
+        backgroundColor: "transparent"
+      }
+    }
+  },
+
+  colorSecondary: {
+    //"#7B303E"
+    "&$checked": {
+      color: "#7B303E",
+      "&:hover": {
+        backgroundColor: "transparent",
+        "@media (hover: none)": {
+          backgroundColor: "transparent"
+        }
+      }
+    },
+    color: "#7B303E",
+    "&:hover": {
+      backgroundColor: "transparent",
+      "@media (hover: none)": {
+        backgroundColor: "transparent"
+      }
+    }
+  },
+
+  checked: {} // Needed to overwrite default behaviour of checked
+});
+
+
+// Two different colors: primary (teal) and secondary (red)
+// Comes with title (label) built in; can be sent in as React Component (e.g., Typography)
+const Tickbox = ({ title, color }) => {
+  const classes = useStyles();
+
+  return (
+    <div className={"checkbox"}>
+      <FormControlLabel
+        control={
+          <Checkbox
+            color={color}
+            classes={{
+              root: classes.root,
+              colorPrimary: classes.colorPrimary,
+              colorSecondary: classes.colorSecondary,
+              checked: classes.checked
+            }}
+          />
+        }
+        label={title}
+      />
+    </div>
+  );
+};
+
+export default Tickbox;
```

Comment:
I know this is not the case with all components so far, but it's less confusing for the component to have the same name as the file. The file's name is checkbox while the component's name is Tickbox.

Reply:
Good point! I just didn't want to get it confused with the MaterialUI checkbox; I'll change both to be Tickbox.